.github/workflows/build.yml

```diff
@@ -14,22 +14,34 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        swift: ["5.6-focal", "5.5-focal", "5.4-focal", "5.3.3-focal", "5.2.5-focal"]
+        swift: ["5.6.2", "5.5.3", "5.4.3", "5.3.3", "5.2.5", "5.1.5", "5.0.3"]
```

Comment:
I was actually wondering why that was the case. Are you fine with keeping the latest `.` versions that I put in or do you want me to revert that?

Reply:
The lack of `x.y.0` was the original reason, and then I realized it meant we didn't have to keep updating them.  The string does go into the cache checks for the protobuf library, but it doesn't seem like anything bad had happened since.

The zlib1g-dev tweak for downloads (which looks like it no longer will be working), was the only other thing that had to be in agreement.

Reply:
I just pushed a fix for the zlib1g-dev tweak.